livespark-form-modeler/livespark-form-modeler-editor/livespark-form-modeler-editor-client/src/main/java/org/livespark/formmodeler/editor/client/editor/rendering/EditorFieldLayoutComponent.java

```diff
@@ -183,13 +187,13 @@ public Modal getConfigurationModal( final ModalConfigurationContext ctx ) {
         return propertiesRenderer.getView().getPropertiesModal();
     }
 
-    @Override
-    public void onDropComponent() {
+    public void onDropComponent(@Observes ComponentDropEvent event) {
+        //TODO convert from ComponentDropEvent for FieldDroppedEvent
         fieldDroppedEvent.fire( new FieldDroppedEvent( formId, fieldId ) );
     }
 
-    @Override
-    public void onRemoveComponent() {
+    public void onRemoveComponent(@Observes ComponentRemovedEvent event) {
+        //TODO convert from ComponentRemovedEvent for FieldDroppedEvent
```

Comment:
@pefernan we have to fix this! could you please point me to the right direction?